### PR TITLE
Update vendor

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by mtrmac** [comment]
+3 million lines, what??!

File: vendor.conf [hunk 1]
```diff
@@ -3,7 +3,7 @@ github.com/urfave/cli v1.20.0
 github.com/kr/pretty v0.1.0
 github.com/kr/text v0.1.0
 github.com/containers/image 2c0349c99af7d90694b3faa0e9bde404d407b145
-github.com/containers/buildah 810efa340ab43753034e2ed08ec290e4abab7e72
+github.com/containers/buildah 1.8.2
```
Comment (mtrmac) [inline vendor.conf]:
```
github.com/containers/buildah: Err: exit status 1, out: error: pathspec '1.8.2' did not match any file(s) known to git.
```
How was this PR created?

Comment (rhatdan) [reply vendor.conf]: make vendor
I will fix it.

**Comment by TomSweeneyRedHat** [comment]
Changes LGTM but a rebase is needed @rhatdan

**Pull request closed without merging**